my_weather.py

```diff
@@ -0,0 +1,39 @@
+'''
+This script prints out the weather for the cities given as arguments.
+Cities should be separated by ','. FE: Dublin,Rome,London
+Optional: add a '-f' flag for Fahrenheit, '-c' flag for Celsius.
+If needed, please change the value of the api key.
+'''
+
+from requests import get
+import json
+import requests
+import sys
+
+# Change api key value to yours if needed
+api_key = 'bc30006e191a5efcc61d86b1db0153bd'
+
+def weather_in(city):
+    city_list = city[1].split(',')
+    fahrenheit = city[-1] == '-f'
```

Comment:
This is an okay solution, but what if I have asked you for another possible flag? let's say a  -h flag that will allow to also print the humidity percentage? how would you implement this now (should the "-h" always be last or maybe "-f"....or maybe both possibilities)?